### ISSUE TITLE
Ability to change ping color

### ISSUE THREAD

**Issue by turt2live** [vector-im/riot-web#6520]
Colorblind people, for instance, can have issues seeing the ping to begin with.

**Comment by turt2live** [comment]
This is arguably a duplicate of https://github.com/vector-im/riot-web/issues/1655 however it'd be nice to be able to change my ping color to lime green if I so desired.

**Comment by lampholder** [comment]
Yeah - let's keep this issue here to represent the desire for customisable ping colour.

**Comment by lampholder** [comment]
I've tweaked some of the labels on https://github.com/vector-im/riot-web/issues/1655 to better reflect problems for colourblind users being a bug (rather than a missing feature)

**Comment by jtrees** [comment]
I think changing the text color on ping is already a bad idea. A different kind of highlight (maybe an icon, a border, etc.) would be better. Please consider a redesign of this feature instead.
